pollux/lib/vc-jwt/src/main/scala/io/iohk/atala/pollux/vc/jwt/revocation/BitString.scala

```diff
@@ -0,0 +1,68 @@
+package io.iohk.atala.pollux.vc.jwt.revocation
+
+import io.iohk.atala.pollux.vc.jwt.revocation.BitStringError.{DecodingError, EncodingError, IndexOutOfBounds}
+import zio.{IO, UIO, ZIO}
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+import java.util
+import java.util.Base64
+import java.util.zip.{GZIPInputStream, GZIPOutputStream}
+
+class BitString private (val bitSet: util.BitSet, val size: Int) {
```

Comment:
Because the size returned by java.util.BitSet, once initialised, is the space used to store the requested number of bits, and not the number of bits itself. I was actually not able to get that initial size out of the BitSet class.

I know this is not perfect but anyway, this constructor is `private`, so not exposed to users and only accessible by the companion object.